src/helpers/getResultStringForAdditionalPropertiesType.ts

```diff
@@ -1,16 +1,14 @@
-import { DataTypes, SwaggerProps } from '../types';
+import { DataTypes, ResultStringForAdditionalPropertiesType, SwaggerProps } from '../types';
```

Comment:
this file was already added by https://github.com/LandrAudio/openapi-codegen-typescript/pull/52 but instead of having two branch for additional properties with and without dictionnary - and duplicate some logic, I added the check here